lib/fastlane/plugin/revenuecat_internal/actions/update_hybrids_versions_file_action.rb

```diff
@@ -19,9 +19,25 @@ def self.run(params)
         ios_version = Helper::UpdateHybridsVersionsFileHelper.get_ios_version_for_hybrid_common_version(hybrid_common_version)
         UI.message("Obtained ios version #{ios_version} for PHC version #{hybrid_common_version}")
 
+        billing_client_version = Helper::UpdateHybridsVersionsFileHelper.get_android_billing_client_version(android_version)
+        UI.message("Obtained android billing client version #{billing_client_version} for PHC version #{hybrid_common_version}")
+
         File.open(versions_file_path, 'r+') do |file|
           lines = file.each_line.to_a
-          lines.insert(2, "| #{new_sdk_version} | #{ios_version} | #{android_version} | #{hybrid_common_version} |\n")
+          if lines[0].split('|').length <= 6
```

Comment:
super magical number here, maybe there's a way to clarify what it means? it was easier to understand before. maybe extracting a boolean + explaining why "6" makes sense, 

i.e: it means we get a column for each one of these: new_sdk_version, ios_version, android_version, hybrid_common_version, billing_client_version + the separator before and after

```ruby
# Determine if the header needs to be extended based on the number of columns
  # The number 6 comes from new_sdk_version, ios_version, android_version, hybrid_common_version, billing_client_version + separators before and after
  extend_header = lines[0].split('|').length <= 6

  if extend_header
```

Reply:
That's... a totally good point 😅 I wanted to extract to a constant before and I forgot. But adding a comment also works 👍